tracks/rust/exercises/leap/README.md

```diff
@@ -0,0 +1,63 @@
+### Reasonable solutions
+
+```rust
+pub fn is_leap_year(year):
+    year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)
+```
+
+### Common suggestions
+- there are just two cases that return True:
+  - a year is a multiple of 4 *and not** 100
```

Comment:
is this supposed to be cursive or bold? It starts with 1 star, ends with 2

Reply:
When I wrote it in the Python version it was meant to be italics.